src/main/java/com/xiaomi/infra/pegasus/client/request/Get.java

```diff
@@ -0,0 +1,36 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+package com.xiaomi.infra.pegasus.client.request;
+
+import java.io.Serializable;
+
+public class Get implements Serializable {
+
+  private static final long serialVersionUID = 5022721155142508979L;
+
+  public final byte[] hashKey;
+  public final byte[] sortKey;
+
+  public Get(byte[] hashKey) {
+    this(hashKey, null);
+  }
```

Comment:
```suggestion
  public Get(byte[] hashKey) {
    this(hashKey, null);
  }
```

I think Get(hashkey) would be misused as getting all the records under the hashkey.